### PR TITLE
Who Govwifi is for page

### PR DESCRIPTION
**Pull request by tommotaylor**
Screenshot 2019-04-25 at 10 14 27

### CODE REVIEW

File: source/who-govwifi-is-for.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Who GovWifi is for
```
Comment (Renny4Real) [inline source/who-govwifi-is-for.html.erb]: Not sure, if it matters but perhaps we should change the title to be `Who GovWifi is for - GovWifi`?
It's the title of the page according to the spreadsheet.

File: source/shared/product/_sidebar.html.erb [hunk 2]
```diff
@@ -1,7 +1,10 @@
 <nav class="sub-navigation">
   <ul>
     <li class="sub-navigation__item">
-      <a href="/what-is-govwifi">What is GovWifi?</a>
+      <%= link_to "What is GovWifi?", "/what-is-govwifi" %>
+    </li>
+    <li class="sub-navigation__item">
+      <%= link_to "Who GovWifi is for", "/who-govwifi-is-for" %>
```
Comment (Renny4Real) [inline source/shared/product/_sidebar.html.erb]: Perhaps this link should be `/about-govwifi/who-govwifi-is-for`?

**Comment by Renny4Real** [comment]
I think this page should be in an `about-govwifi` folder?

File: source/who-govwifi-is-for.html.erb (hunk 1 above)
Comment (tommotaylor) [reply source/who-govwifi-is-for.html.erb]: Nice catch

File: source/shared/product/_sidebar.html.erb (hunk 2 above)
Comment (tommotaylor) [reply source/shared/product/_sidebar.html.erb]: You're right. It looks like we've made this mistake elsewhere too. I've fixed up the `What is GovWifi?"` /  `"/about-govwifi"` page too.

**Pull request merged**